examples/models/unet.py

```diff
@@ -0,0 +1,148 @@
+import torch.nn as nn
+from models.modules.layers_3d import (SparseConvBlock, SparseDeConvBlock,
+                                      SparseResBlock)
+
+import torchsparse
+import torchsparse.nn as spnn
+
+__all__ = ['SparseResUNet']
+
+
+class SparseResUNet(nn.Module):
+
+    def __init__(self, **kwargs):
+        super().__init__()
+
+        cr = kwargs.get('cr', 1.0)
+        cs = [32, 32, 64, 128, 256, 256, 128, 96, 96]
```

Comment:
We should avoid hard-code the channel size in the model. We could make it an argument in `__init__`.